app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -77,4 +79,10 @@ public static int getNextNonWhitespace(CharSequence s, int start, int maxRange)
         return selection;
     }
 
+    public static String repeatChars(Character character, int count) {
```

Comment:
suggest to use char, and convert to Character inside if required - it's impossible to get null exception this way